operations/update.go

```diff
@@ -102,6 +111,19 @@ func Update() cli.Command {
 					return err
 				}
 				grip.Info("Upgrade complete!")
+
+				if runtime.GOOS == "windows" {
+					grip.Infoln("Deleting old binary", winTempDest)
+					// Source: https://stackoverflow.com/a/19748576
+					// Since Windows does not allow a binary that's currently in
+					// use to be deleted, wait 2 seconds for the CLI process to
+					// exit and then delete it.
+					cmd := exec.Command("cmd", "/c", fmt.Sprintf("ping localhost -n 3 > nul & del %s", winTempDest))
```

Comment:
I don't think this will work, as the running process has a lock on the old binary (even after the rename). I think you have to delete this when the evergreen binary is next run, or dump it in the system temp folder and yolo it.

Reply:
I tried this on a Windows spawn host and it seemed to work fine. I didn't rename it to the temp dir because `os.Rename()` doesn't work across drives and Windows spawn hosts have their temp folder in `Z:/` instead of `C:/`.